modules/social_features/social_profile/social_profile.install

```diff
@@ -472,7 +472,7 @@ function social_profile_update_11701(): string {
  * - "view profile",
  * - "add any profile profile".
  */
-function social_profile_update_111101(): void {
```

Comment:
This change should land on 11.x as it it BC. Hook update number should not be changed.